static/js/protein/page.tsx

```diff
@@ -19,22 +19,76 @@
  */
 
 import React from "react";
+import axios from "axios";
+
+import { GraphNode } from "../shared/types";
 
 interface PagePropType {
   dcid: string;
   nodeName: string;
 }
 
-export class Page extends React.Component<PagePropType> {
+interface PageStateType {
+  data: GraphNode;
+}
+
+export class Page extends React.Component<PagePropType, PageStateType> {
   constructor(props: PagePropType) {
     super(props);
+    this.state = { data: null };
   }
 
   componentDidMount(): void {
-    // TOOD: add data fetching here.
+    this.fetchData();
   }
 
   render(): JSX.Element {
-    return <div> Protein Data</div>;
+    const tissueScore = this.getTissueScore();
+    // TODO: use d3 to draw bar chart here.
```

Comment:
@suhana13 @khoahoang1891999 This will be where you call the d3 library. We have some examples like https://github.com/datacommonsorg/website/blob/master/static/js/chart/draw_choropleth.ts#L113.

You can create a similar function to draw the new chart and call it in this file.